src/libmscore/measure.h

```diff
@@ -186,10 +186,12 @@ class Measure final : public MeasureBase
     bool isIrregular() const { return m_timesig != _len; }
 
     int size() const { return m_segments.size(); }
-    Segment* first() const { return m_segments.first(); }
+    Ms::Segment* first() const { return m_segments.first(); }
```

Comment:
It should be already in Ms namespace

Reply:
I want just port, without any changes, see https://github.com/musescore/MuseScore/blob/3.x/libmscore/measure.h#L149